### PR TITLE
Add totals program

### PR DESCRIPTION
**Pull request by lindsaymarkward**
Please review this work @judahward - check naming, etc.

### CODE REVIEW

File: totals.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""
+Total income program
+"""
+
+def main():
+    """Display income report for incomes over a given number of months."""
```
Comment (judahward) [inline totals.py:6]: Nice docstring

File: totals.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    incomes = []
+    months = int(input("How many months? "))
```
Comment (judahward) [inline totals.py:8]: This name seems plural, like it might be a list...

File: totals.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    for month in range(1, months + 1):
+        income = float(input(f"Enter income for month {month}: "))
+        incomes.append(income)
+    print_report(incomes, months)
+
+
+def print_report(incomes, number):
```
Comment (judahward) [inline totals.py:15]: Missing docstring

File: totals.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    print("\nIncome Report\n-------------")
+    total = 0
+    for month in range(1, number + 1):
```
Comment (judahward) [inline totals.py:18]: If you used the length of incomes, you shouldn't need this parameter.

**Review by judahward — COMMENTED** [review]
Nice work. A few small changes 👍 🌮

File: totals.py (hunk 2 above)
Comment (lindsaymarkward) [reply totals.py:8]: Good point. 👈

**Pull request merged**